Testing/VelaptorTests/NativeInterop/OpenGL/OpenGLServiceTests.cs

```diff
@@ -135,7 +160,7 @@ public void SetViewPortSize_WhenInvoked_SetsViewPort()
         service.SetViewPortSize(new Size(55, 66));
 
         // Assert
-        this.mockGLInvoker.Verify(m => m.Viewport(11, 22, 55, 66));
+        this.mockGLInvoker.Received().Viewport(11, 22, 55, 66);
```

Comment:
Please refactor the `Received()` method to take the parameter value of `1`.

```suggestion
        this.mockGLInvoker.Received(1).Viewport(11, 22, 55, 66);
```